dashboard/db/migrate/20160617163547_create_user_geos.rb

```diff
@@ -0,0 +1,16 @@
+class CreateUserGeos < ActiveRecord::Migration
+  def change
+    create_table :user_geos do |t|
+      t.references :user, index: true, unique: true, foreign_key: true, null: false
+      t.integer :user_id
+      t.timestamps null: false
+      t.datetime :indexed_at, null: false
+      t.string :ip_address
```

Comment:
Do we want a `ip_address_at` timestamp, indicating the timestamp at which we obtained the `ip_address` from the user?